src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -5,25 +5,44 @@ import java.util.UUID
 
 import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
 import org.broadinstitute.dsde.workbench.leonardo.model.ClusterStatus.ClusterStatus
-import org.broadinstitute.dsde.workbench.leonardo.model.ModelTypes.{GoogleBucket, GoogleProject, GoogleServiceAccount}
+import org.broadinstitute.dsde.workbench.leonardo.model.LabelMap.LabelMap
 import spray.json.{DefaultJsonProtocol, DeserializationException, JsString, JsValue, JsonFormat}
 
-// maybe we want to get fancy later
-object ModelTypes {
-  type GoogleProject = String
-  type GoogleServiceAccount = String
-  type GoogleBucket = String
+sealed trait TypedString { val value: String }
```

Comment:
I am unable to get this to work.  Tests are failing with errors like `ClusterName(ClusterName(test)) does not equal ClusterName(test)`